Source/Krypton Components/Krypton.Toolkit/General/OutlookGrid/General/OutlookGridGroupCountComparer.cs

```diff
@@ -29,15 +29,15 @@ public OutlookGridGroupCountComparer()
 
         #region IComparer Members
 
-        public int Compare(IOutlookGridGroup x, IOutlookGridGroup y)
+        public int Compare(IOutlookGridGroup? x, IOutlookGridGroup? y)
```

Comment:
`x` nullable ignored, restore and use `DisallowNull` to allow compiler to check.

Reply:
Changed

Reply:
Ok - go for it and put some comments in the code to state why please .

Reply:
Changed in the upcoming commit.